commands/cfconfig/componentpath/save.cfc

```diff
@@ -0,0 +1,86 @@
+/**
+* Add a new Component Path
+* 
+* {code}
+* cfconfig componentpath save /foo C:/foo/bar
+* cfconfig componentpath save virtual=/foo physical=C:/foo/bar to=serverName
+* cfconfig componentpath save virtual=/foo physical=C:/foo/bar to=/path/to/server/home
+* {code}
+*
+*/
+component {
+	
+	property name='CFConfigService' inject='CFConfigService@cfconfig-services';
+	property name='Util' inject='util@commandbox-cfconfig';
+	property name="serverService" inject="ServerService";
+	
+	/**
+	* @name The name of the path component path
+	* @physical The physical path that the engine should search
+	* @archive Path to the Lucee/Railo archive
+	* @inspectTemplate String containing one of "never", "once", "always", "" (inherit)
+	* @inspectTemplate.options never,once,always
+	* @primary Strings containing one of "physical", "archive"
+	* @primary.options physical,archive
+	* @readonly true/false
+	* @to CommandBox server name, server home path, or CFConfig JSON file. Defaults to CommandBox server in CWD.
+	* @to.optionsFileComplete true
+	* @to.optionsUDF serverNameComplete
+	* @toFormat The format to write to. Ex: LuceeServer@5
+	*/	
+	function run(
+		string name,
+		string physical="",
```

Comment:
The same rules for defaulting non-required fields applies here.  I never default anything at this level and only at the last minute when writing the XML.  This ensures the JSON only reflects attributes which were explicitly provided.  All I think we need to do is take of the default and adjust the validation below to account for the possibility of these being null.